Specs/Number/Arabic/NumberModel.json

```diff
@@ -1738,8 +4345,9 @@
   },
   {
     "Input": "1 في واحد و عشرون",
-    "NotSupported": "javascript, python",
+    "Comment": "PendingValidation",
     "NotSupportedByDesign": "javascript,python,java",
+    "NotSupported": "javascript, python, dotnet",
```

Comment:
Minor issue: should be only dotnet, as other are already in NotSupportedByDesign

Reply:
These were existing test cases from the older spec. We have added dotnet to NotSupported as part of the PR.  A total of 9 such cases were found in NumberModel and we have fixed it. A new pull request 2083 is raised where these changes are included.